content/applications/inventory_and_mrp/manufacturing/management/work_order_dependencies.rst

```diff
@@ -0,0 +1,118 @@
+=======================
+Work order dependencies
+=======================
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+
+When manufacturing certain products, specific operations may need to be completed before others can
+begin. In order to ensure operations are carried out in the correct order, Odoo *Manufacturing*
+features a *work order dependencies* setting. Enabling this setting allows for operations on a Bill
+of Materials (BoM) to be *blocked* by other operations that should occur first.
+
+Configuration
+=============
+
+The *work order dependencies* setting is not enabled by default. To enable it, begin by navigating
+to :menuselection:`Manufacturing --> Configuration --> Settings`. Enable the :guilabel:`Work Orders`
+setting if it is not already active.
+
+After enabling the :guilabel:`Work Orders` setting, the :guilabel:`Work Order Dependencies` setting
+appears below it. Enable :guilabel:`Work Order Dependencies`, then click :guilabel:`Save` to confirm
+the changes.
+
+Add dependencies to a BoM
+=========================
+
+Work order dependencies are configured on a product's |BOM|. To do so, navigate to
+:menuselection:`Manufacturing --> Products --> Bills of Materials`, then select a |BOM|, or create a
+new one by clicking :guilabel:`New`.
+
+.. admonition:: Learn more
+
+   For a complete guide on how to properly configure a new |BOM|, see the documentation on
+   :ref:`creating a bill of materials <manufacturing/management/bill-configuration>`.
+
+On the |BOM|, click on the :guilabel:`Miscellaneous` tab, then enable the :guilabel:`Operation
+Dependencies` checkbox. This makes a new :guilabel:`Blocked By` option available in the settings for
+the :guilabel:`Operations` tab.
+
+.. image:: work_order_dependencies/operation-dependencies.png
+   :align: center
+   :alt: The Operation Dependencies checkbox on the Miscellaneous tab of a BoM.
+
+Next, click on the :guilabel:`Operations` tab. On the top right of the tab, click on the tab's
+:guilabel:`settings` button, then enable the :guilabel:`Blocked By` checkbox. This makes a
+:guilabel:`Blocked By` field appear for each operation on the :guilabel:`Operations` tab.
+
+.. image:: work_order_dependencies/operations-settings.png
+   :align: center
+   :alt: The settings for the Operations tab on a BoM.
+
+In the line of the operation that should be blocked by another operation, click the
+:guilabel:`Blocked By` field, and a :guilabel:`Open: Operations` pop-up window appears. In the
+:guilabel:`Blocked By` drop-down field on the pop-up window, select the operation that must be
+completed before the operation that is blocked.
```

Comment:
I know what's happening in this paragraph because I'm familiar with how blocked operations work, but it's a little hard to tell which operation is going first. My only idea is using an example, which might not be ideal because we want to make our instructions as general as possible. How about something like:

```suggestion
To block the `Assemble table` operation with the `Cut table legs` operation, for example, click on
the :guilabel:`Blocked By` field for `Assemble table`. Doing so opens the :guilabel:`Open:
Operations` pop-up. In the :guilabel:`Blocked By` drop-down field on the pop-up window, select `Cut
table legs`.
```

Reply:
Just read your line 114, and I really like the way you described "... leading from the blocking operation to the blocked operation". Maybe we can change the wording here to specify "blocking operation" and "blocked operation" instead of the more general "operation.